modules/planning/tasks/deciders/path_bounds_decider.cc

```diff
@@ -123,15 +124,15 @@ Status PathBoundsDecider::Process(
   return Status::OK();
 }
 
-std::string PathBoundsDecider::GeneratePathBoundaries(
+std::string PathBoundsDecider::GenerateRegularPathBoundary(
     Frame* frame, ReferenceLineInfo* reference_line_info,
     std::vector<std::tuple<double, double, double>>* const path_boundaries) {
   // Sanity checks.
   CHECK_NOTNULL(frame);
   CHECK_NOTNULL(reference_line_info);
 
   // 1. Initialize the path boundaries to be an indefinitely large area.
-  if (!InitPathBoundaries(reference_line_info->reference_line(),
+  if (!InitPathBoundary(reference_line_info->reference_line(),
                           frame->PlanningStartPoint(), path_boundaries)) {
```

Comment:
Fix alignment when you changed corresponding code.

Reply:
fixed alignments.